drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -7,9 +7,25 @@
 class Hmis::Hud::Client < Hmis::Hud::Base
   include ::HmisStructure::Client
   include ::Hmis::Hud::Shared
+  include ArelHelper
+
   self.table_name = :Client
   self.sequence_name = "public.\"#{table_name}_id_seq\""
 
+  scope :hmis_scope, ->(user) do
```

Comment:
Let's rename this `visible_to`

Reply:
```suggestion
  scope : visible_to, ->(user) do
    joins(:data_source).merge(GrdaWarehouse::DataSource.hmis.where(id: user.hmis_data_source_id))
```
Or maybe make `user` a parameter to the `hmis` scope so we can get the HMIS record for the specific user.